image-loader/app/controllers/ImageLoaderController.scala

```diff
@@ -136,9 +137,8 @@ class ImageLoaderController(auth: Authentication,
           respondError(NotFound, "image-not-found", s"Could not find image: $imageId in s3 at $s3Path")
       } recover {
         case _: NoSuchImageExistsInS3 => NotFound(Json.obj("imageId" -> imageId))
-        case _ => InternalServerError(Json.obj("imageId" -> imageId))
+        case e => InternalServerError(Json.obj("imageId" -> imageId, "exception" -> e.getMessage))
```

Comment:
I think there is a standard format for errors, although it seems to be fairly poorly applied so perhaps there should be one but currently isn't. I note that `getMessage` loses a lot of context such as the exception type (super useful for context) and stack trace (nice to have).